### ISSUE TITLE
openstack_lb_member_v2 optional subnet_id

### ISSUE THREAD

**Issue by pblaas** [terraform-providers/terraform-provider-openstack#188]
Hi,

I would like to propose for the subnet_id in the openstack_lb_member_v2 resource to be optional:

https://github.com/terraform-providers/terraform-provider-openstack/blob/master/openstack/resource_openstack_lb_member_v2.go
```
"subnet_id": &schema.Schema{
  Type:     schema.TypeString,
  Required: true,
  ForceNew: true,
}
```
### Terraform Version
Terraform v0.11.1
+ provider.openstack v1.0.0

### Affected Resource(s)
- openstack_lb_member_v2

### Expected Behavior
subnet_id  in the  openstack_lb_member_v2 resource should be optional.

### Actual Behavior
subnet_id  in the openstack_lb_member_v2 resources is required but not saved by Opencontrail and therefor the resource is destroyed and created on every apply of the Terraform config. This will drop traffic to the members of the loadbalancer.

### Steps to Reproduce
1. `create a loadbalancer setup`
2. `terraform apply`
3. `terraform apply` 

### Important Factoids
Tested on Openstack Mikata with OpenContrail.

### References
Contrail not supporting subnet_id:
http://www.opencontrail.org/documentation/api/r3.2/contrail_openapi.html#loadbalancermembertype-label

Neutron models spec subnet_id for loadbalancer pool members should NULL-able: 
https://github.com/openstack/neutron-lbaas/blob/master/neutron_lbaas/db/loadbalancer/models.py#L88

**Comment by jtopjian** [comment]
@pblaas Thank you for pointing this out. I believe you're correct here.

I've opened #189 which will resolve this.

**Comment by pblaas** [comment]
Great! thank you for following up on this.

Do i need to close this feature request or is it automaticly closed after applying the fix in #189.

**Comment by jtopjian** [comment]
It'll automatically close once #189 is merged. That's preferable so we ensure this issue is resolved.
